IFC4x3/Sections/Domain specific data schemas/Schemas/IfcPortsAndWaterways/Entities/IfcLiquidTerminal/DocEntity.xml

```diff
@@ -2,7 +2,9 @@
 <DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcLiquidTerminal" UniqueId="f042a74a-1026-46d7-9a2d-5e48dea59687" BaseDefinition="IfcFlowTerminal" EntityFlags="32">
 	<Attributes>
 		<DocAttribute Name="PredefinedType" UniqueId="332bc28d-be4e-487e-bacb-c2738f652f63" DefinedType="IfcLiquidTerminalTypeEnum" AttributeFlags="1">
-			<Documentation></Documentation>
+			<Documentation>Identifies the predefined type of a liquid terminal from which the type modelled, may be set. This type may associate additional specific property sets.
+NOTE  The PredefinedType shall only be used, if no _IfcLiquidTerminalType_  is assigned, providing its own IfcLiquidTerminalType.PredefinedType.
```

Comment:
Should be handled in WR 'CorrectPredefinedType'.

Reply:
well it is a common note on lots of other PDT attributes so i keep it as a easy reminder to implementers and readers...